diff --git a/node_modules/trtc-react-native/android/build.gradle b/node_modules/trtc-react-native/android/build.gradle
index 7afe4f3..0596ec4 100644
--- a/node_modules/trtc-react-native/android/build.gradle
+++ b/node_modules/trtc-react-native/android/build.gradle
@@ -86,7 +86,7 @@ dependencies {
   // For < 0.71, this will be from the local maven repo
   // For > 0.71, this will be replaced by `com.facebook.react:react-android:$version` by react gradle plugin
   //noinspection GradleDynamicVersion
-  api 'com.tencent.liteav:LiteAVSDK_Professional:12.8.+'
+  api 'com.tencent.liteav:LiteAVSDK_Professional:12.8.0.19279'
   implementation 'com.google.code.gson:gson:2.8.6'
   implementation "com.facebook.react:react-native:+"
 }
